### ISSUE TITLE
BeginEndMarkReceiveFilter不能容错

### ISSUE THREAD

**Issue by coocoin** [kerryjiang/SuperSocket.ProtoBase#1]
在使用EasyClient中BeginEndMarkReceiveFilter时，如`<end>eeeeeerror<begin>`这种内容会导致连接关闭，是否能添加容错能正确解析出`<begin>Message<end>errrrr<begin>Message<end>`这种消息

**Comment by kerryjiang** [comment]
Yes, you can. Just implement the receive filter by yourself completely.
